### DISCUSSION TITLE
considering const qualifiers for mutability rules

### DISCUSSION THREAD

**Discussion by arhik** [JuliaInterop/Clang.jl#491] [Ideas]
I would like to initiate a conversation on this idea of considering `const` qualifiers for `automutability` configuration option. Since this is potentially breaking change initiating discussion on it and then working on it would be ideal to save man hours. Once agreeing on the rules we can accept PR along those lines and also redirect people who want to work on this to this discussion.

This was originally raised issue here https://github.com/JuliaInterop/Clang.jl/issues/490

**Comment by Gnimuc** [comment]
The `automutability` option is experimental, so any PRs are welcome. :) 

We can document those changes if needed.
